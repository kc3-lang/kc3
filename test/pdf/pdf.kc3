quote require PDF
require PDF
quote PDF.parse_str("%Hello, PDF !\n")
PDF.parse_str("%Hello, PDF !\n")
quote PDF.parse_str("123\n")
PDF.parse_str("123\n")
quote PDF.parse_str("(abc)\n")
PDF.parse_str("(abc)\n")
quote PDF.parse_str("(ab(())c)\n")
PDF.parse_str("(ab(())c)\n")
quote PDF.parse_str("<41424344>\n")
PDF.parse_str("<41424344>\n")
