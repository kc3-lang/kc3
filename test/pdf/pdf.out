require PDF
true
PDF.parse_str("%Hello, PDF !\n")
void
PDF.parse_str("123\n")
void
PDF.parse_str("(abc)\n")
"abc"
PDF.parse_str("(ab(())c)\n")
"ab(())c"
PDF.parse_str("<41424344>\n")
"ABCD"
